vscode-dotnet-sdk-extension/package.json

```diff
@@ -59,7 +59,7 @@
 				},
 				"dotnetSDKAcquisitionExtension.installTimeoutValue": {
 					"type": "number",
-					"default": 240,
+					"default": 300,
```

Comment:
Interesting, I guess slower networks bring this into play? Out of curiosity is the 1min increase based on a specific metric or a magic number?

Reply:
No specific metric, just seems to make sense to me